packages/react-ui-rent/src/LeadModal/CloseButton.js

```diff
@@ -34,6 +36,9 @@ export default class CloseButton extends PureComponent {
           theme.CloseButton
         )}
         {...props}
+        onClick={onClick}
```

Comment:
You don't need to pass this. The whole point of react-ui-core Modal is so you don't have to do this. See: https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Modal/ModalBody.js#L41. Passing props through here accomplishes what you need.